.github/workflows/vroom.yml

```diff
@@ -0,0 +1,26 @@
+name: Vroom CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
It would be great to avoid triggering a build when only markdown files are changed. Looks like `paths-ignore` would do that?

The typical situation is a single commit to just add a changelog entry.

Reply:
Added markdown files(.md) to ignore.